design/cdn-edge.md

```diff
@@ -6,62 +6,62 @@ On this diagram the arrow points in the direction of who instantiated the reques
 
-Layer 1 servers will connect to 2 unique layer 2 servers. This is to provide redundancy if incase a layer 2 server goes down. 
+Layer 1 servers will connect to 2 unique Layer 2 servers. This is to provide redundancy if a Layer 2 server goes down.
 
 ## Layer 2
 
-Layer 2 servers will act as a 2nd cache layer and will relay instructions to the layer 1 servers recieved from the control servers.
+Layer 2 servers will act as a 2nd cache Layer and relay instructions to the Layer 1 servers received from the control servers.
 
-Layer 2 servers will connect to one layer 2 server in each region (ie, if we have 3x layer 2 server in NA and 3x layer 2 servers, each layer 2 server in NA will connect to 1 layer 2 server in EU and vice-versa)
+Layer 2 servers will connect to one Layer 2 server in each region (i.e., if we have 3x Layer 2 server in NA and 3x Layer 2 servers, each Layer 2 server in NA will connect to 1 Layer 2 server in EU and vice-versa)
 
-Layer 2 will also act as a router. If we recieve a request from a layer 1 server to go to an origin and the layer 2 server does not have the data. If the layer 2 server can fetch the data from an origin it will be routed to said origin, if it cannot then the request will be routed to the layer 2 server which has an origin attached and then the origin will recieve the request. 
+Layer 2 will also act as a router. If we receive a request from a Layer 1 server to go to an origin and the Layer 2 server does not have the data. If the Layer 2 server can fetch the data from an origin, it will be routed to the origin. If it cannot, then the request will be routed to the Layer 2 server, which has an origin attached, and then the origin will receive the request.
 
-In terms of network hops.
+In terms of network Hops.
 - Layer 1 (cache miss)
 - Layer 2 (cache miss)
-- Layer 2 (cache miss & no direct connecton)
+- Layer 2 (cache miss & no direct connection)
 - Origin (source of truth)
 
-If layer 2 in hop 2 has a direct connection to the origin we will route directly to hop 4 (skipping over hop 3).
+If Layer 2 in Hop 2 directly connects to the origin, we will route directly to Hop 4 (skipping over Hop 3).
 
 ## Control
 
-The control servers will act as the state of the system. They will contain all the information about what routes exist in the network. They will also act as a authentication and state management server for the layer 2 servers which will in turn act as a manager for layer 1 servers.
+The control servers will act as the state of the system. They will contain all the information about what routes exist in the network. They will also serve as an authentication and state management server for the Layer 2 servers, which will act as a manager for Layer 1 servers.
 
-Control servers are completely isolated from the request lifecycle and therefore do not a large bandwidth pipe.
+Control servers are entirely isolated from the request lifecycle and therefore do not have a large bandwidth pipe.
```

Comment:
Is "entirely" really the right word here? "completely" sounded pretty on point to me :thinking:

Reply:
entirely is fine perhaps instead of `have` we can use `need`